### PR TITLE
Grpc.Tools document AdditionalImportDirs

### PR DESCRIPTION
**Pull request by tonydnewell**
Add documentation for AdditionalImportDirs
See https://github.com/grpc/grpc/issues/22878

### CODE REVIEW

File: src/csharp/BUILD-INTEGRATION.md [hunk 1]
```diff
@@ -389,12 +398,17 @@ and "GrpcServices" metadata, but it might be useful in situations where you want
 to explicitly pass some otherwise unsupported (e.g. experimental) options to the
 `grpc_csharp_plugin`.
 
+* __AdditionalImportDirs__
+Pass additional directories to include proto files.  These may be directories _outside_ of the
```
Comment (jtattermusch) [inline src/csharp/BUILD-INTEGRATION.md:402]: IMHO it's inaccurate to say "Pass additional directories to include proto files".
How about "Specify additional directories in which the protocol buffer compiler will search for imported proto files."?

FTR, the official helptext for the `-I` flag in `protoc` is : *Specify the directory in which to search for imports.  May be specified multiple times; directories will be searched in order. ...*

File: src/csharp/BUILD-INTEGRATION.md [hunk 2]
```diff
@@ -352,6 +360,7 @@ The following metadata are recognized on the `<Protobuf>` items.
 | GrpcOutputDir  | See notes | A directory                        | Directory for generated gRPC stubs    |
 | GrpcOutputOptions | | arbitrary options                  | Extra options passed to gRPC codegen as `--grpc_opt=opt1,opt2` |
 | GrpcServices   | `both`    | `none`, `client`, `server`, `both` | Generated gRPC stubs             |
+| AdditionalImportDirs | See notes | Directories                        | Additional directories for to find files |
```
Comment (jtattermusch) [inline src/csharp/BUILD-INTEGRATION.md:363]: typo: "Additional directories for to find files"? Let's update to something more intelligible (also see the comment below).

File: src/csharp/BUILD-INTEGRATION.md [hunk 3]
```diff
@@ -91,6 +91,14 @@ button](https://stackoverflow.com/a/9770061). Click on it, and choose "Add as
 link". If you do not select this option, Visual Studio will copy files to the
 project directory instead.
 
+Alternatively, you can also specify `AdditionalImportDirs` to give the location of proto files outside
```
Comment (jtattermusch) [inline src/csharp/BUILD-INTEGRATION.md:94]: rephrase "to give the location of proto files outside
the project" as e.g. "to provide the list of directories in which the protocol buffer compiler will search for imported proto files".
(your original phrasing seems a bit vague).

**Review by jtattermusch — COMMENTED** [review]
Left a few comments to address, but otherwise looks good.

File: src/csharp/BUILD-INTEGRATION.md (hunk 1 above)
Comment (tonydnewell) [reply src/csharp/BUILD-INTEGRATION.md]: updated

File: src/csharp/BUILD-INTEGRATION.md (hunk 2 above)
Comment (tonydnewell) [reply src/csharp/BUILD-INTEGRATION.md]: updated

File: src/csharp/BUILD-INTEGRATION.md (hunk 3 above)
Comment (tonydnewell) [reply src/csharp/BUILD-INTEGRATION.md]: updated

**Review by jtattermusch — APPROVED** [review]
LGTM.

**Comment by jtattermusch** [comment]
This PR is simply a change in docs, so test failures are unrelated.

**Comment by jtattermusch** [comment]
I fixed a typo. I will merge once the test finish.

**Pull request merged**